App.js

```diff
@@ -1,82 +1,19 @@
-/**
-* Sample React Native App
-* https://github.com/facebook/react-native
-*
-* @format
-* @flow strict-local
-*/
+import * as React from 'react';
+import {NavigationContainer} from '@react-navigation/native';
+import {createStackNavigator} from '@react-navigation/stack';
+import Login from './Login.js'
+const Stack = createStackNavigator();
 
-import React from 'react';
-import { StyleSheet, Button, View, SafeAreaView, Text, Alert, TouchableOpacity, ImageBackground } from 'react-native';
-
-function Separator() {
-  return <View style={styles.separator} />;
-}
-
-const image = { url: '/Users/virginialadd/projects/wet-tits/wetTits/ios/assets/abs-bg.jpg' };
-
-export default function App() {
+function MyStack() {
   return (
-    <SafeAreaView style={ styles.container }>
-      <ImageBackground source={ image }  style={ styles.image } >
-      <View>
-        <TouchableOpacity style={ styles.buttonTop } >
-          <Text style={{ fontSize: 32, color: 'lightgray' }}>Train Today</Text>
-        </TouchableOpacity>
-      </View>
-      <Separator />
-      <View>
-        <TouchableOpacity style={ styles.buttonBottom } >
-          <Text style={{ fontSize: 32, color: 'lightgray' }}>Run Today</Text>
-        </TouchableOpacity>
-      </View>
-      </ImageBackground>
-    </SafeAreaView>
+    <NavigationContainer>
+      <Stack.Navigator>
```

Comment:
This is super cool to look at! Are the components pre set?

Reply:
Components are from an NPM dependency yes